proknow-sdk/IProKnowApi.cs

```diff
@@ -47,6 +47,11 @@ public interface IProKnowApi
         /// </summary>
         Requestor Requestor { get; }
 
+        /// <summary>
+        /// Issues requests to the RTV API
+        /// </summary>
+        RtvRequestor RtvRequestor { get; }

```

Comment:
I don't think this should be exposed to users

Reply:
FYI, it's exposed in the Python SDK (just like the normal Requestor) so that users can easily make a request manually to a route that may not yet be implemented by the SDK.